docs/api/StreamOpen.md

```diff
@@ -21,15 +21,43 @@ QUIC_STATUS
 
 # Parameters
 
-**TODO**
+`Connection`
+
+The valid handle to an open connection object. The connection does **not** need to be started via [ConnectionStart](ConnectionStart.md).
+
+`Flags`
+
+A set of flags that control the behavior of `StreamOpen`:
+
+Value | Meaning
+--- | ---
+**QUIC_STREAM_OPEN_FLAG_NONE**<br>0 | No special behavior. Defaults to bidirectional stream.
+**QUIC_STREAM_OPEN_FLAG_UNIDIRECTIONAL**<br>1 | Opens a unidirectional stream.
+**QUIC_STREAM_OPEN_FLAG_0_RTT**<br>2 | Indicates that the stream may be sent in 0-RTT.
+
+`Handler`
+
+A pointer to the app's callback handler to be invoked for all stream events.
+
+`Context`
+
+The app context pointer (possibly null) to be associated with the stream object.
```

Comment:
You might add something like "this context will be passed to the user's handler when invoked"